ospfd/ospf_dump.c

```diff
@@ -1503,6 +1505,33 @@ DEFUN (no_debug_ospf_te,
   return CMD_SUCCESS;
 }
 
+DEFUN (debug_ospf_trace,
```

Comment:
I specifically asked for this to be rethought and put into something like debug ospf vrf.